README.md

```diff
@@ -114,16 +114,16 @@ Under the `maps` section, map layers are associated with data provider layers an
 # note mvt data providers can not be conflated with any other providers of any type in a map
 # thus a map may only contain a single mvt provider.
 [[providers]]
-name = "my_postgis"         # provider name is referenced from map layers (required). 
+name = "my_postgis"         # provider name is referenced from map layers (required).
 type = "mvt_postgis"        # the type of data provider must be "mvt_postgis" for this data provider (required)
 uri = "postgresql://tegola:<password>@localhost:5432/tegola?ssl_mode=prefer" # database connection string
 
   [[providers.layers]]
   name = "landuse"
   # MVT data provider must use SQL statements
   # this table uses "geom" for the geometry_fieldname and "gid" for the id_fieldname so they don't need to be configured
-  # Wrapping the geom with ST_AsMVTGeom is required. 
-  sql = "SELECT ST_AsMVTGeom(geom,!BBOX!) AS geom, gid FROM gis.landuse WHERE geom && !BBOX!"
+  # Wrapping the geom with ST_AsMVTGeom is required.
+  sql = "SELECT ST_AsMVTGeom(geom,!BBOX!) AS geom, gid FROM gis.landuse WHERE geom && !BBOX! !PARAM!"
```

Comment:
What do you think about carving the `!PARAM!` token into a separate example? That way we can introduce the concept to the user after they have the `!BBOX!` token working.

Reply:
You are right, that makes sense. I'll add another commit.

Reply:
No worries, I hope this will be ok.

Reply:
@ARolek I forgot the --force and merge was also commited. Please let me know if I should squash this again.